From 221b70faead52ac5ed748b47f76725ad72c8b022 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 25 May 2020 15:22:50 +0000
Subject: [PATCH 06/13] Make SCons inherit environment vars

---
 SConstruct | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SConstruct b/SConstruct
index 075e97911f..a42fd23240 100644
--- a/SConstruct
+++ b/SConstruct
@@ -1127,6 +1127,7 @@ if get_option('build-tools') == 'next' or get_option('ninja') == 'next':
     SCons.Tool.DefaultToolpath.insert(0, os.path.abspath('site_scons/site_tools/next'))
 
 env = Environment(variables=env_vars, **envDict)
+env.Append(ENV = os.environ)
 
 # Only print the spinner if stdout is a tty
 if sys.stdout.isatty():
-- 
2.26.2

