From 79cae3e3d3f5e9de5182a7bd63021f45d0b676e6 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 25 May 2020 19:21:32 +0000
Subject: [PATCH 09/13] Use platform specific TimeStamp implementation

---
 src/third_party/mozjs-60/SConscript | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/src/third_party/mozjs-60/SConscript b/src/third_party/mozjs-60/SConscript
index 42741bc85e..e0326c0e45 100644
--- a/src/third_party/mozjs-60/SConscript
+++ b/src/third_party/mozjs-60/SConscript
@@ -121,9 +121,15 @@ else:
     sources.extend([
     "extract/mozglue/misc/ConditionVariable_posix.cpp",
     "extract/mozglue/misc/Mutex_posix.cpp",
-    "extract/mozglue/misc/TimeStamp_posix.cpp",
     ])
-
+    if env.TargetOSIs('darwin'):
+        sources.extend([
+            "extract/mozglue/misc/TimeStamp_darwin.cpp"
+        ])
+    else:
+        sources.extend([
+            "extract/mozglue/misc/TimeStamp_posix.cpp"
+        ])
 
 
 sources.append( [ "extract/modules/fdlibm/{}".format(f) for f in [
-- 
2.26.2

