From 91e392b68ec09a4644af51d668b633d293df80cf Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 25 May 2020 19:50:43 +0000
Subject: [PATCH 10/13] Fix openssl 1.1.x library names on Windows

---
 SConstruct | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SConstruct b/SConstruct
index c97911270d..043b478642 100644
--- a/SConstruct
+++ b/SConstruct
@@ -3248,9 +3248,9 @@ def doConfigure(myenv):
             sslLinkDependencies = ["crypto"]
 
         if conf.env.TargetOSIs('windows'):
-            sslLibName = "ssleay32"
-            cryptoLibName = "libeay32"
-            sslLinkDependencies = ["libeay32"]
+            sslLibName = "libssl"
+            cryptoLibName = "libcrypto"
+            sslLinkDependencies = ["libcrypto"]
 
         # Used to import system certificate keychains
         if conf.env.TargetOSIs('darwin'):
-- 
2.26.2

