From 50ab3c8f8dd2627ea9d858c7b0db4371a47a56cf Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Tue, 26 May 2020 14:11:37 +0000
Subject: [PATCH 12/13] Don't force disable auto linking if --use-system-boost

---
 SConstruct | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/SConstruct b/SConstruct
index 7fb3ccde68..00b81adc7f 100644
--- a/SConstruct
+++ b/SConstruct
@@ -1783,11 +1783,12 @@ elif env.TargetOSIs('windows'):
         print("NOTE: Tool configuration did not find 'cl' compiler, falling back to os environment")
         env['ENV'] = dict(os.environ)
 
-    env.Append(CPPDEFINES=[
-    # This tells the Windows compiler not to link against the .lib files
-    # and to use boost as a bunch of header-only libraries
-        "BOOST_ALL_NO_LIB",
-    ])
+    if not use_system_version_of_library("boost"):
+        env.Append(CPPDEFINES=[
+        # This tells the Windows compiler not to link against the .lib files
+        # and to use boost as a bunch of header-only libraries
+            "BOOST_ALL_NO_LIB",
+        ])
 
     env.Append( CPPDEFINES=[ "_UNICODE" ] )
     env.Append( CPPDEFINES=[ "UNICODE" ] )
-- 
2.26.2

