From b8a83edb6f2c8c7f781516d264784aef14f76aaa Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nwani@tower-research.com>
Date: Sun, 27 Sep 2020 00:36:16 -0400
Subject: [PATCH 14/14] Relax constraint on minimum version of Visual Studio
 Compiler

---
 SConstruct | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SConstruct b/SConstruct
index 89c044ab78..60c5a30259 100644
--- a/SConstruct
+++ b/SConstruct
@@ -2197,14 +2197,14 @@ def doConfigure(myenv):
     # bare compilers, and we should re-check at the very end that TryCompile and TryLink still
     # work with the flags we have selected.
     if myenv.ToolchainIs('msvc'):
-        compiler_minimum_string = "Microsoft Visual Studio 2019 16.4"
+        compiler_minimum_string = "Microsoft Visual Studio 2017 15.9"
         compiler_test_body = textwrap.dedent(
         """
         #if !defined(_MSC_VER)
         #error
         #endif
 
-        #if _MSC_VER < 1924
+        #if _MSC_VER < 1916
         #error %s or newer is required to build MongoDB
         #endif
 
-- 
2.17.1

